docsite/index.html

```diff
@@ -264,28 +264,28 @@ <h2>Why use Open Props?</h2>
               <path d="M17.657 18.657A8 8 0 016.343 7.343S7 9 9 10c0-2 .5-5 2.986-7C14 5 16.09 5.777 17.656 7.343A7.975 7.975 0 0120 13a7.975 7.975 0 01-2.343 5.657z" />
               <path d="M9.879 16.121A3 3 0 1012.015 11L11 14H9c0 .768.293 1.536.879 2.121z" />
             </svg>
-            Design Consistently
+            Consistent Design
           </dt>
-          <dd>Incidentally harmonious.</dd>
+          <dd>Open Props is incidentally harmonious. It Just Works.</dd>
         </div>
         <div>
           <dt>
             <svg viewBox="0 0 24 24">
               <path d="M4.318 6.318a4.5 4.5 0 000 6.364L12 20.364l7.682-7.682a4.5 4.5 0 00-6.364-6.364L12 7.636l-1.318-1.318a4.5 4.5 0 00-6.364 0z" />
             </svg>
-            Predictable Props
+            Predictable
           </dt>
-          <dd>Small naming pattern.</dd>
+          <dd>Thanks to consistent naming conventions.</dd>
         </div>
         <div>
           <dt>
             <svg  viewBox="0 0 24 24" >
               <path d="M11 3.055A9.001 9.001 0 1020.945 13H11V3.055z" />
               <path d="M20.488 9H15V3.512A9.025 9.025 0 0120.488 9z" />
             </svg>
-            Easy Adoption
+            Incrementally Adoptable
           </dt>
-          <dd>Grab a few props, all the props, props as JS, or adopt only what you use.</dd>
+          <dd>Grab all the props, props as JS, or only what you need.</dd>
```

Comment:
I also had a question about this point:

> Map the props from JS or customize builds from the command line

I think it's worth clarifying the wording here. When we say `map the props from JS`, does this mean you can access Open Props's custom properties in JavaScript as variables?

Reply:
yep! all the props are importable as js